near/omni-types/src/prover_args.rs

```diff
@@ -15,6 +15,8 @@ pub struct VerifyProofArgs {
 pub struct EvmVerifyProofArgs {
     pub proof_kind: ProofKind,
     pub proof: EvmProof,
+    pub min_header_height: Option<u64>,
+    pub max_header_height: Option<u64>,
```

Comment:
It doesn't make sense to pass these arguments by the sender; the connector contract should pass them, so we need to move them somewhere upper or remove them and relay on the nonce instead.

Reply:
Just removed it b8e29324908172ad23993bb641b0acb214c69434